### PR TITLE
Translate latest news (id)

### PR DESCRIPTION
**Pull request by meisyal**
I just translated the latest news on Ruby programing language website into Bahasa Indonesia. I have a question, does TBD mean *to be determined*?
It appears [here](https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md).

Please, review my translation @gozali!
I don't forget to mention @stomar for making some reviews too.
Thank you.

### CODE REVIEW

**Comment by stomar** [comment]
Can this be merged?

**Comment by stomar** [comment]
@meisyal I really tend to merge these, instead of having them lie around for another couple of months...

So in case you agree I suggest you rebase the branch onto current master and I will merge then.

**Comment by meisyal** [comment]
You can rebase this. But, how about waiting @gozali to review this translation?

**Comment by stomar** [comment]
He seems to be busy. I'm rather confident you did all right, so I'm fine with merging. Unless you have some doubts about the translations...

File: id/news/_posts/2015-12-25-ruby-2-3-0-released.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+opsi *command line* `--debug=frozen-string-literal`.
+
+Sebuah [*safe navigation operator*](https://bugs.ruby-lang.org/issues/11537)
+([disebut *lonely operator*](https://instagram.com/p/-M9l6mRPLR/)) `&.`,
+yang telah ada di C#, Groovy, dan Swift, diperkenalkan untuk memudahkan
+penanganan `nil` sebagai `obj&.foo`. `Array#dig` dan `Hash#dig` juga ditambahkan.
+Catat bahwa ini berperilaku sebagai [*try! of Active Support*](http://api.rubyonrails.org/v4.2.5/classes/Object.html#method-i-try-21),
+yang khusus menangani *nil*.
+
+[*did_you_mean gem is bundled*](https://bugs.ruby-lang.org/issues/11252).
```
Comment (stomar) [inline id/news/_posts/2015-12-25-ruby-2-3-0-released.md]: untranslated

Comment (meisyal) [inline id/news/_posts/2015-12-25-ruby-2-3-0-released.md]: Which word or sentence that have not  been translated?

Comment (stomar) [inline id/news/_posts/2015-12-25-ruby-2-3-0-released.md]: "did_you_mean gem is bundled"

Comment (meisyal) [inline id/news/_posts/2015-12-25-ruby-2-3-0-released.md]: Oh, you're right. I will translate and push a commit later.

**Comment by meisyal** [comment]
I just fixed that post translation. Should I squash commits?

**Comment by stomar** [comment]
:+1: Squash would be appreciated; also a rebase onto current master.

**Comment by meisyal** [comment]
Done. I have squashed commits and re-based onto current master. You may check it out.

**Comment by stomar** [comment]
:clap: Thanks!

**Comment by meisyal** [comment]
You're welcome :)

**Pull request merged**